operatorhub/Makefile

```diff
@@ -0,0 +1,12 @@
+BUNDLE_ARGS := $(or ${BUNDLE_ARGS},--help)
+
+OPERATOR_SDK = $(or ${OPERATOR_SDK_BIN},$(BIN)/operator-sdk)
+
+$(BIN)/operator-sdk: | $(BIN) ; $(info $(M) getting operator-sdk)
+	ls $(BIN)
+	@./operatorhub/tools/install_operator-sdk.sh $(BIN)
```

Comment:
Is this path right ? 🤔

Reply:
as.. we are in `operatorhub` folder already, so if I run the `Makefile` while being in `operatorhub` folder, it is gonna fail isn't it ?

Reply:
we are using `include operatorhub/Makefile` in the main Makefile. So we are not using a `make -c` in the call to `operator-bundle`. The working dir will be repository root.